### PR TITLE
Login screen

### LINKED ISSUE #6: Login Screen
- [ ] Make a login screen with input fields 

ESTIMATE: 3

### PR DESCRIPTION
**Pull request by filipfloden**
Fixes #6 

# Description

This PR implements the login screen which will be used to login users when we have access the API for it.

## Lists of changes

- [x] Overall design
- [x] TextField for email input
- [x] SecureField for password input
- [x] Log in button
- [x] Forgot password text which is meant to be embedded in a NavigationLink when the Recover password page is made.

image

## Testing

Short summary of what should be tested
- [ ] See if the email & password fields placeholder works. Should be moved to overlap the top border
- [ ] See if the design looks fine on different size devices

### CODE REVIEW

**Review by Yevve — CHANGES_REQUESTED** [review]
Add a test account validation for login so it can be used in transition testing.

File: FlexiCharge/FlexiCharge/Views/Login/LoginView.swift [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+            }
+            // Login "form"
+            VStack {
+                // Email input field
+                ZStack(alignment: .topLeading) {
+                    TextField(emailPlaceholder, text: $emailInput)
+                        .frame(height: inputHeight)
+                        .offset(x: 8)
+                        .overlay(RoundedRectangle(cornerRadius: inputCornerRadius).stroke())
+                    Text(emailPlaceholder)
```
Comment (scamdivine) [inline FlexiCharge/FlexiCharge/Views/Login/LoginView.swift:39]: one .something on each line plzy

**Review by scamdivine — APPROVED** [review]

**Comment by filipfloden** [comment]
> Add a test account validation for login so it can be used in transition testing.

Daniel suggested that. we just make the login button redirect to the startpage when we combine all the register/login views. I think we should go with this idea!

**Review by Yevve — APPROVED** [review]
LGTM!

**Pull request merged**